src/components/Popover.tsx

```diff
@@ -98,7 +99,7 @@ export default function Popover({
   contained,
   showArrow = true,
 }: PopoverProps) {
-  const boundary = useContext(BoundaryContext)
+  const { boundary, updateTrigger } = useContext(BoundaryContext) || {}
```

Comment:
can you declare an `const EMPTY_OBJECT = {}` in a util file and use that instead so its not a new obj on every render 🤔 not a huge deal here since this value itself isnt used butttt more of a habit thing

Reply:
I gave the context a default value of {} instead